browser/brave_stats/brave_stats_updater_params.cc

```diff
@@ -219,7 +224,7 @@ GURL BraveStatsUpdaterParams::GetUpdateURL(
   update_url =
       net::AppendQueryParameter(update_url, "arch", GetProcessArchParam());
   update_url =
-      net::AppendQueryParameter(update_url, "wallet", GetWalletEnabledParam());
+      net::AppendQueryParameter(update_url, "wallet2", GetWalletEnabledParam());
```

Comment:
Renamed the wallet param in the usage ping so we can filter out the erroneous data and start over for this month. Thoughts? cc @bbondy

Reply:
I'm cool with it. 
great name :P